### ISSUE TITLE
`vel_start` and `vel_end` in `computePathParametrization()` handled differently

### ISSUE THREAD

**Issue by mskripnik** [hungpham2511/toppra#255]
```
PathParametrizationAlgorithm::computePathParametrization(value_type vel_start, value_type vel_end)
```

`vel_start` is the path velocity squared at s=0
`vel_end` is the actual path velocity at s=1

This is not only not documented, but also very misleading.

**Comment by mishrasubhransu** [comment]
Yes. I had to change `m_data.parametrization(0) = vel_start;` to `m_data.parametrization(0) = vel_start * vel_start;` to make both of them path velocity.

**Comment by christophfroehlich** [comment]
@mishrasubhransu you mean here?
https://github.com/hungpham2511/toppra/blob/6b11235b304455a944479f1cb250559f1eb44b7b/cpp/src/toppra/algorithm/toppra.cpp#L22
